modules/materials-page/materials-page.service.js

```diff
@@ -0,0 +1,50 @@
+const Materials = require('./materials-page.model');
+const createTranslations = require('../../utils/createTranslations');
+const {
+  addTranslations,
+  updateTranslations,
+  deleteTranslations,
+} = require('../translations/translations.service');
+
+class MaterialsService {
+  async getAllMaterialsBlocks() {
+    return Materials.find({}).exec();
+  }
+
+  async addMaterialsBlock(materialsBlock) {
```

Comment:
I would add here some check for existence of such material. It's migth be proper to add getMaterialBlockById method and throw error in case of it exists

Reply:
fix)